app/assets/stylesheets/govuk_publishing_components/components/_cookie-banner.scss

```diff
@@ -24,3 +26,70 @@ $govuk-cookie-banner-background: #d5e8f3;
     display: inline-block;
   }
 }
+
+// Styles for the new version of the cookie banner
+.gem-c-cookie-banner--new {
+  @include govuk-font($size: 16);
+  border: 2px solid govuk-colour("black");
+}
+
+.gem-c-cookie-banner__wrapper.js-hidden {
```

Comment:
I tend to prefer using a modifier for these (`.gem-c-cookie-banner__wrapper--hidden`), but we don't have a style guide to refer to so it's up to you for now. Another way of doing it is to rely on JavaScript for `style.display= 'none'` to be consistent with the `$cookieBannerConfirmationMessage`.